presto-mongodb/pom.xml

```diff
@@ -230,9 +230,6 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
-                            <includes>
```

Comment:
This profile is named `test-mongo-distributed-queries`. If we don't want to run this class separately, we can either remove this profile and use only the default profile in tests, or keep this profile as is and enable the default profile separately in tests as well for other tests.

https://github.com/agrawalreetika/prestodb/blob/2092534f0aa5261c79c97fcaf5d784357d20f80f/presto-mongodb/pom.xml#L205-L238